src/main/registry.ts

```diff
@@ -446,6 +468,16 @@ export class Registry implements IRegistry, IDisposable {
     return this._requirements;
   }
 
+  getRequirementsPipInstallCommand(): string {
```

Comment:
I am afraid that suggesting `pip` install will lead to many problems if novice conda users end up executing this in their env. Worst case scenario is having two conflicting versions with assets from one and scripts form another which is quite common problem but hard to debug.

Reply:
is this an issue when an old jupyterlab is installed with `conda install`, then updated with `pip install`. I thought pip install was fine in a conda environment.

Reply:
It will only work when NOT in the base environment (ideally in isolated environment) and if `pip` was explicitly installed. Many users fall into this trap.

Reference: https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#using-pip-in-an-environment

Reply:
Of course if you install the same version in base and leaf env, one using pip and the other using conda, it does not matter. Up until the very moment one gets updated and then it breaks.